consensus/consensus-types/src/block.rs

```diff
@@ -274,6 +275,10 @@ impl Block {
         }
     }
 
+    pub fn sys_txns(&self) -> Option<&Vec<SystemTransaction>> {
```

Comment:
`sys_txn` -> `system_txn` everywhere?

Reply:
tried, the diff will be too big... prefer doing it later in another PR.